R/OsModel.R

```diff
@@ -0,0 +1,118 @@
+#' Title - TODO
+#'
+#' Description - TODO
+#'
+#' @slot stan TODO
+#' @slot templated Logical. Defines whether the OsModel object needs to be completed
+#' @export
+OsModel <- setClass(
+    Class = "OsModel",
+    representation = list(
+        stan = "StanModule"
+    )
+)
+
+
+
+#' LogLogisticModule
+#'
+#' Log logistic module helper function.
```

Comment:
```suggestion
#' Log logistic module helper function.
#'
```
similar in other places for visual readability